homeassistant/components/iotawatt/__init__.py

```diff
@@ -0,0 +1,130 @@
+"""The iotawatt integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+from httpx import AsyncClient
+from iotawattpy.iotawatt import Iotawatt
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_SCAN_INTERVAL,
+    CONF_USERNAME,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import (
+    COORDINATOR,
+    DEFAULT_ICON,
+    DEFAULT_SCAN_INTERVAL,
+    DOMAIN,
+    SIGNAL_ADD_DEVICE,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up iotawatt from a config entry."""
+    polling_interval = entry.options.get(CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL)
```

Comment:
This integration does not have an option flow, so this will never exist, just  set `DEFAULT_SCAN_INTERVAL` directly on line 83  (super call to coordinator)

Reply:
Removed the setting of the polling interval and set the ```DEFAULT_SCAN_INTERVAL``` in the ```super()`` call.